stable-2.4.0.md

```diff
@@ -0,0 +1,188 @@
+## stable-2.4.0
+
+This stable release introduces adds support for SMI [Traffic Split](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md),
+fine-grained observability into the TLS-based identity system via the new
+`linkerd edges` command, default automatic proxy injection, installation stages,
+and a host of performance and usability improvements in the proxy and Web UI.
+
+Creating an SMI [Traffic Split](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md)
+resource will allow Linkerd to split traffic between the specified backend
+services. The `linkerd edges` command provides a thorough breakdown of the name
+and identity for proxied connections, as well as an exciting new "Edges" table
+in the Web UI!
+
+Significant changes to the installation lifecycle have introduced default
+automatic proxy injection, and optional installation stages via the `linkerd
+install config` and `linkerd install control-plane` commands which separate the
+concerns for cluster-level and namespace-level privileges.
+
+Among the many performance and usability improvements to the proxy and Web UI,
+debugging efforts have been accompanied by a new a new `--enable-debug-sidecar`
+flag for the `linkerd inject` command and more accurate metric collection of
+proxied connections.
+
+TODO: Update blog post link. For more details, see the announcement [blog post](https://linkerd.io/blog/)
+
+To install this release, run: `curl https://run.linkerd.io/install | sh`
+
+**Upgrade notes**: TODO: Add summary of upgrade notes link. Please see the
```

Comment:
When https://github.com/linkerd/website/pull/396 is merged and published, the upgrade notes will be at https://linkerd.io/2/tasks/upgrade/#upgrade-notice-stable-2-4-0